src/app/http/http.service.ts

```diff
@@ -1,25 +1,20 @@
 import { Injectable } from '@angular/core';
 import { Observable, of } from 'rxjs';
+import { Movie } from '../interfaces';
 
 @Injectable({
   providedIn: 'root'
 })
 export class HttpService {
-  private movie: newMovie[] = [];
+  private movie: Movie[] = [];
```

Comment:
1. to jest array filmów więc movies, nie movie 2. nie trzymaj filmów w http service. Http service posłuży Ci później o zapytania o dane z youtube/vimeo, na razie nic tu nie rób, wyczyść to w ogóle. Filmy będą trzymane w storze (state)

Reply:
także ja ten komponent bym wyczyścił w ogóle na razie i zostawił w spokoju

Reply:
ok dzięki